test/scenarios/attestation/configuration/readme.az.md

```diff
@@ -7,8 +7,10 @@ az:
   extensions: attestation
   package-name: azure-mgmt-attestation
   namespace: azure.mgmt.attestation
+  use-test-step-param: true
 az-output-folder: $(azure-cli-extension-folder)/src/attestation
 python-sdk-output-folder: "$(az-output-folder)/azext_attestation/vendored_sdks/attestation"
+gen-cmdlet-test: true
```

Comment:
Could we use a relatively complex RP for this scenario test? just don't think this is a good candidate to test some corner case such as polymorphism or complex resource dependencies

Reply:
added datafactory